platform-sdk/sdk/settings.txt

```diff
@@ -14,4 +14,5 @@ useLoopbackIp,                    false
 csvFileName,                      PlatformTesting
 checkSignedStateFromDisk,         true
 loadKeysFromPfxFiles,             false
-event.preconsensus.enableStorage, false
\ No newline at end of file
+event.preconsensus.enableStorage, false
+reconnect.active,                 true
```

Comment:
thank you!

Reply:
At this point in time I'm not sure why it doesn't default to `true`... the main reason I've not changed it is because the process for changing configuration is not well defined.